source/agora/consensus/Quorum.d

```diff
@@ -66,32 +66,31 @@ public struct QuorumParams
     to the quorum assignment of each node.
 
     Params:
-        key = the key of the node for which to generate the quorum
+        utxo_key = the key of the node for which to generate the quorum
         utxo_keys = the list of UTXO keys of all the active enrollments
         finder = UTXO finder delegate
         rand_seed = the random seed
         params = quorum generator algorithm tweaking parameters
 
 *******************************************************************************/
 
-public QuorumConfig buildQuorumConfig (in PublicKey key,
-    in Hash[] utxo_keys, scope UTXOFinder finder, in Hash rand_seed,
-    const ref QuorumParams params)
+public QuorumConfig buildQuorumConfig (in Hash utxo_key, in Hash[] utxo_keys,
```

Comment:
Honestly I don't know if the rename is necessary. We don't consistently use the terms anywhere. `utxo_key` feels weird, as it repeats the type name, essentially (`Hash`).

Reply:
Right. The original meaning for the `key` was `the key of the node for which to generate the quorum` so the name should remain as it was. Will revert.